src/jfrMetadata.cpp

```diff
@@ -202,6 +202,14 @@ JfrMetadata::JfrMetadata() : Element("root") {
                 << field("heapSpace", T_VIRTUAL_SPACE, "VirtualSpace")
                 << field("heapUsed", T_LONG, "Heap Used", F_BYTES))
 
+            << (type("jdk.MethodTrace", T_METHOD_TRACE, "Method trace")
+                << category("Java Virtual Machine", "Profiling")
```

Comment:
Let event label and category match [OpenJDK declarations](https://github.com/openjdk/jdk/blob/991ac9e6168b2573f78772e2d7936792a43fe336/src/jdk.jfr/share/classes/jdk/jfr/events/MethodTraceEvent.java#L34-L35).
```
@Label("Method Trace")
@Category({ "Java Virtual Machine", "Method Tracing" })
```

Reply:
+ Capital `T` in the label

Reply:
1b9328f0cf02cce230ad01572a3dff69ecbae28a